samples/MvcSample.Web/Startup.cs

```diff
@@ -107,6 +107,19 @@ public void Configure(IApplicationBuilder app)
                         options.Filters.Add(typeof(PassThroughAttribute), order: 17);
                         options.AddXmlDataContractSerializerFormatter();
                     });
+                    services.ConfigureAuthorization(auth =>
+                    {
+                        auth.AddPolicy("CanViewPage",
```

Comment:
nit: formatting : all parameters should be on top of one another. 
something like 
```c#
    auth.AddPolicy(
        "CanViewAnything",
        new AuthorizationPolicyBuilder().RequiresClaim("Permission", "CanViewAnything")
            .Build());
```